SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -255,6 +255,7 @@ const FString ProjectPattern    = TEXT("^[a-z0-9_]{3,32}$");
 const FString ProjectPatternHint = TEXT("Project name may only contain lowercase alphanumeric characters or '_', and must be between 3 and 32 characters long.");
 const FString DeploymentPattern = TEXT("^[a-z0-9_]{2,32}$");
 const FString DeploymentPatternHint = TEXT("Deployment name may only contain lowercase alphanumeric characters or '_', and must be between 2 and 32 characters long.");
+const FString Ipv4Pattern = TEXT("^(?:[0-9]{1,3}\\.){3}[0-9]{1,3}$");
```

Comment:
Question, would we want to support Options added to IPs like this?
For example `127.0.0.1?MyOption`
If so we should look into adjusting this regex.
Currently we use `const FURL URL(nullptr /* Base */, *URLAddress, TRAVEL_Absolute);` in `ConnectionConfig` for this feature.

Reply:
We might in the future, though I would expect those to be specified via a separate field when we want to support that use case.

Reply:
Additionally, port?

Reply:
Modifying the receptionist port is not something we want to expose here. The new flow doesn't do anything about the port so if it was possible to change that before they can still do that.

Reply:
Sure, but isn't this the regex used to control which receptionist you connect too? If someone on LAN has started the runtime on another port (I believe @jherd-improbable did this recently) then you'd want to be able to connect to said runtime still yah?

Reply:
It's a separate feature that hasn't been done yet. The logic that's in this PR is only using this regex to match an IP, passing the port is not supported.